Source/items.cpp

```diff
@@ -4147,11 +4133,11 @@ int RndSmithItem(int lvl)
 
 void BubbleSwapItem(ItemStruct *a, ItemStruct *b)
 {
-	ItemStruct h; // [esp+8h] [ebp-170h]
+	ItemStruct h;
 
-	qmemcpy(&h, a, sizeof(h));
-	qmemcpy(a, b, sizeof(ItemStruct));
-	qmemcpy(b, &h, sizeof(ItemStruct));
+	h = *a;
+	*a = *b;
+	*b = h;

```

Comment:
Haha, nice cleanup.

Reply:
:) This even has a duplicate in stores.cpp